guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc

```diff
@@ -19,22 +19,15 @@ For more information, see the {AdministeringDocURL}Installing_the_Katello_Root_C
 .Procedure
 . On the VM host system, configure the firewall to allow VNC service on ports 5900 to 5930:
 +
-ifdef::satellite[]
-* On {RHEL} 6:
-endif::[]
 ifndef::satellite[]
 * On operating systems with the `iptables` command:
 endif::[]
```

Comment:
You should hide the `iptables` instructions on `satellite
```suggestion
```

Reply:
Addressed :+1: